### PR TITLE
Sort last date

### PR DESCRIPTION
**Pull request by pepperrone**

### CODE REVIEW

File: front-end/src/components/toolbar/SortingBar.js [hunk 1]
```diff
@@ -7,17 +7,15 @@ export function SortingBar(props) {
...
-          value={1}
-        >
+        <ToggleButton variant="light" onClick={props.onClick} prechecked value={1}>
           Sort by number of mentees
         </ToggleButton>
         <ToggleButton variant="light" onClick={props.onClick1} value={2}>
           Sort by mentor
         </ToggleButton>
+        <ToggleButton variant="light" onClick={props.onClick2} value={3}>
+          Sort by last met
```
Comment (edtheshed) [inline front-end/src/components/toolbar/SortingBar.js]: "Sort by last meeting date" i think might be better

File: front-end/src/util/sorting.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
+export function sortByLastMetDate(leftCraftsperson, rightCraftsperson) {
```
Comment (edtheshed) [inline front-end/src/util/sorting.js]: i think everywhere it is always referred to as lastMeeting, I would rename all of these to that

File: front-end/src/components/toolbar/SortingBar.js (hunk 1 above)
Comment (edtheshed) [reply front-end/src/components/toolbar/SortingBar.js]: Or actually, we could bring in line more with the mockup, where we have a label:
"Sort by:"
Then each button..

Comment (pepperrone) [reply front-end/src/components/toolbar/SortingBar.js]: done

File: front-end/src/util/sorting.js (hunk 2 above)
Comment (pepperrone) [reply front-end/src/util/sorting.js]: done

**Pull request merged**